internal/pkg/template/templates/environment/cf.yml

```diff
@@ -97,24 +97,60 @@ Resources:
           Value: disabled
           {{- end}}
 {{- end}}
-  PublicLoadBalancerSecurityGroup:
+  PublicHTTPLoadBalancerSecurityGroup:
     Metadata:
-      'aws:copilot:description': 'A security group for your load balancer allowing HTTP and HTTPS traffic'
+      'aws:copilot:description': 'A security group for your load balancer allowing HTTP traffic'
     Condition: CreateALB
     Type: AWS::EC2::SecurityGroup
     Properties:
-      GroupDescription: Access to the public facing load balancer
+      GroupDescription: HTTP access to the public facing load balancer
       SecurityGroupIngress:
+        {{- if .PublicFacingCIDRPrefixListIDs}}
+        {{- range $id := .PublicFacingCIDRPrefixListIDs}}
+        - SourcePrefixListId: {{$id}}
+          Description: Allow ingress from prefix list {{$id}} on port 80
+          FromPort: 80
+          IpProtocol: tcp
+          ToPort: 80
+        {{- end}}
+        {{- else}}
         - CidrIp: 0.0.0.0/0
           Description: Allow from anyone on port 80
           FromPort: 80
           IpProtocol: tcp
           ToPort: 80
+        {{- end}}
+{{- if .VPCConfig.Imported}}
+      VpcId: {{.VPCConfig.Imported.ID}}
+{{- else}}
+      VpcId: !Ref VPC
+{{- end}}
+      Tags:
+        - Key: Name
+          Value: !Sub 'copilot-${AppName}-${EnvironmentName}-lb'
+  PublicHTTPSLoadBalancerSecurityGroup:
```

Comment:
qq: I see we are trying to add this ingress only if the ALB is listening for HTTPS on 443. This makes sense. Any other reasons why we are doing this now?

Reply:
Splitting the security group is needed because one security group has a max count of prefix lists of 60, but each cloud front prefix list counts for 55 of those, so in order to still only allow http and https traffic, we need to split them into two different SGs.

Reply:
To add on @CaptainCarpensir. One other benefit is that we don't always open HTTPS 443 for users anymore if they don't enable https.